ch.protonmail.protonmail-bridge.appdata.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>ch.protonmail.protonmail-bridge</id>
+  <name>Protonmail Bridge</name>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
Do we have redistribution rights, given it's proprietary? If not, manifest needs to be changed to use `extra-data`.